### DISCUSSION TITLE
Reproducability report Access Denied

### DISCUSSION THREAD

**Discussion by Risho92** [jishnujayakumar/MLRC2020-EmbedKGQA#5] [Q&A]
@jishnujayakumar , I was trying to access the reproducability report. But I am getting access denied error.

https://openreview.net/pdf?id=mqMG-MDWOpS

{"status":400,"name":"forbidden","message":"Forbidden","errors":[{"type":"forbidden","path":"id","value":"mqMG-MDWOpS","user":"guest_1612054663320"}]}

**Comment by jishnujayakumar** [comment]
It's submitted for peer review. I feel that's the reason for this error. Did you try logging in?

**Reply by Risho92** [reply]
I did not log into openreview.net. I tried to create a credential, but it is asking for a school/work email id and a profile page. I don't have a profile page. So I will wait for the peer review to be complete for you to publish the numbers in this github page. Generally how much time will it take for peer review to complete?

**Reply by jishnujayakumar** [reply]
I think that it should be done by mid March.

**Reply by jishnujayakumar** [reply]
Please visit https://rescience.github.io/bibliography/P_2021.html
